### PR TITLE
Contenu dynamique

### PR DESCRIPTION
**Pull request by RMayeux**

### CODE REVIEW

**Comment by clemcx** [comment]
ok approved

**Review by clemcx — APPROVED** [review]
ok approved

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -2,22 +2,76 @@
...
+        }
+        $demands = $demandRepository ->findAll();
+        $tags = $tagRepository->findAll();
+
+        /* Création d'une demande */
+        $demand = new Demand();
+        $form = $this->createForm(DemandType::class, $demand);
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager = $this->getDoctrine()->getManager();
```
Comment (benj-san) [inline src/Controller/AdminController.php]: Could you put the entity manager as a parameter of the function

**Review by benj-san — COMMENTED** [review]
Fix the comment and it is ok.
Please, no more PR like that.
Atomic is the key.
Also I can see some migrations files belonging (I hope) to Clemence, but I cannot be sure of it because this merge comes from an "all commit" branch.
Stop this branch once this PR is on dev.

**Review by benj-san — APPROVED** [review]

**Pull request merged**